pack.go

```diff
@@ -90,7 +119,60 @@ func Pack(ctx context.Context, pusher content.Pusher, layers []ocispec.Descripto
 		Size:      int64(len(manifestBytes)),
 	}
 
-	// push manifest.
+	// push manifest
+	if err := pusher.Push(ctx, manifestDesc, bytes.NewReader(manifestBytes)); err != nil && !errors.Is(err, errdef.ErrAlreadyExists) {
+		return ocispec.Descriptor{}, fmt.Errorf("failed to push manifest: %w", err)
+	}
+
+	return manifestDesc, nil
+}
+
+// PackArtifact packs the given blobs, generates an ORAS Artifact Manifest for
+// the pack, and pushes it to a content storage.
+// If succeeded, returns a descriptor of the manifest.
+// Returns ErrMissingArtifactType if artifactType is empty.
+// Reference: https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md
+func PackArtifact(ctx context.Context, pusher content.Pusher, blobs []artifactspec.Descriptor, artifactType string, opts PackArtifactOptions) (ocispec.Descriptor, error) {
```

Comment:
Just want to confirm that the OCI store target could be used as the `pusher` here? I believe the interface is implemented but want to double check.

Reply:
Yes, OCI store implements `Pusher` (`Pusher` is a part of `Target`)